neqo-http3/src/features/extended_connect/tests/webtransport/mod.rs

```diff
@@ -189,10 +190,18 @@ impl WtTest {
     }
 
     fn exchange_packets(&mut self) {
+        const RTT: Duration = Duration::from_millis(10);
         let mut out = None;
+        let mut now = now();
         loop {
-            out = self.client.process(out, now()).dgram();
-            out = self.server.process(out, now()).dgram();
+            now += RTT / 2;
+            out = self.client.process(out, now).dgram();
+            now += RTT / 2;
+            out = self.server.process(out, now).dgram();
+            now += RTT / 2;
+            out = self.client.process(out, now).dgram();
+            now += RTT / 2;
+            out = self.server.process(out, now).dgram();
```

Comment:
Can you add some explanation as to why you need two iterations (client->server->client->server, then loop) rather than one (client->server, loop)?  It's not obvious to me why this is necessary.

There's another version of this that looks like this:

```rust
            out = self.client.process(out, now()).dgram();
            let client_none = out.is_none();
            out = self.server.process(out, now()).dgram();
            if client_none && out.is_none() {
                break;
            }
```
That might be clearer (and less likely to be wasteful).

Reply:
I'll need to dig into this. I can only tell that one iteration does not work for now.